(DOCSP-26998): @realm/react-ify Bundle a Realm (#2596)
mongodb/docs-realm: source/sdk/react-native/realm-database/bundle.txt

On a high level you:

> #. Create a bundled realm file in a Node.js script.

Reviewer: Having this overview list feels redundant when we have a nice procedure directive right after it. I think we could drop "On a high level you:" and the ordered list that precedes the procedure directive.

#. Copy the bundled realm to the iOS and/or Android builds for your application.
#. Call ``Realm.copyBundledRealmFiles()``  before you open the bundled Realm in
   your React Native JavaScript code.

.. TODO: add back in DOCSP-28176
.. .. note:: Bundle Synchronized Realms

..    SDK version 10.12.0 introduced the ability to bundle synchronized realms.
..    Before version 10.12.0, you could only bundle local realms.

..    For details on considerations and limitations when bundling a synchronized realm,
..    :ref:`see the Bundle a Synchronized Realm section <react-native-bundle-synced-realm>`.

.. _react-native-create-a-realm-for-bundling:

.. procedure::

   .. step:: Create a Realm File to Bundle

      The easiest way to create a bundled realm for your React Native app is

>       to write a separate Node.Js script to create the bundle.

Reviewer 2: ```suggestion
      to write a separate Node.js script to create the bundle.
```

>       You can use the ``realm`` package to create your bundle rather than ``@realm/react``.

Reviewer: We should be more direct. There isn't a way to do it with `@realm/react`, right?
```suggestion
      Don't use the ``realm`` package to create your bundle.
      ``@realm/react`` does not have a way to create bundles.
```

Author: you could, there's just no reason to do so, and it'd be overly complicated.

Author: maybe change 'can' to 'should'?

Author: ```suggestion
      You should use the ``realm`` package to create your bundle rather than ``@realm/react``.
```
